annotation_store/tests.py

```diff
@@ -168,34 +169,6 @@ def test_lti_passback_triggered(self):
         mock_store.lti_grade_passback()
         mock_store.lti_grade_passback.assert_called_with()
 
-    # Ensuring no grade passback if not assignment or if teacher
-    @mock.patch.object(ims_lti_py.tool_provider.DjangoToolProvider, 'post_replace_result')
-    def test_lti_passback_not_triggered(self, mock_post_replace_result):
```

Comment:
Removed because of `ims_lti_py` (deprecated). Should ensure that this is being tested still.